### ISSUE TITLE
I am Using ngx-owl-carousel, our client requirement is two rows sliding. is it possible can you please help on this 

### ISSUE THREAD

**Issue by ramanjiram** [mujtaba01/ngx-owl-carousel#52]
Our client requirement is like below.can anyone help on this

![slider](https://user-images.githubusercontent.com/29566742/42720907-0b275bee-874e-11e8-9119-aee957accbcb.jpg)

**Comment by dfjs** [comment]
@ramanjiram yes what you're looking to do _is_ possible using Owl Carousel 2, but GitHub issues isn't really the best place to ask this kind of question, as it's not an issue, as such. 

I'd recommend asking on StackOverflow about using Owl Carousel 2, specifically: 

https://stackoverflow.com/questions/tagged/owl-carousel-2

Good luck!
